custom/download_qrda.php

```diff
@@ -150,10 +150,10 @@ function closeMe() {
         <th scope="col" class="multiDownload">
             <input type="checkbox" name="checkAll" id="checkAll"/>
             <div style="display:none" id=downloadAll>
-                <img class='downloadIcon' src='<?php echo $GLOBALS['webroot'];?>/images/downbtn.gif' onclick=downloadAllXML(); />
+                <img class="downloadIcon" src="<?php echo $GLOBALS['onclick=downloadAllXML()'];?>"/>
             </div>
-            <div style='display:none' id=spinAll>;
-                <img src='<?php echo $GLOBALS['webroot'];?>/interface/pic/ajax-loader.gif'/>
+            <div style="display:none" id=spinAll>
+                <img src="<?php echo $GLOBALS['images_static_relative'];?>/interface/pic/ajax-loader.gif"/>
```

Comment:
this doesn't look right. Shouldn't there just be /ajax-loader.gif rather than /interface/pic/ajax-loader.gif ?

Reply:
The $GLOBALS variable leads to $web_root/public/images, but this gif is in interface/pics, not public/images. Wouldn't that lead to an incorrect path?

Reply:
Then would revert this line back to original:
```
<img src='<?php echo $GLOBALS['webroot'];?>/interface/pic/ajax-loader.gif'/>
```